configs/openmrs_config/ampathforms/cervical_cancer_screening.json

```diff
@@ -764,10 +764,10 @@
 					},
 					{
 					"isExpanded": "true",
-					"label": "Cervical Biopsis",
```

Comment:
@kazlaw, why are we changing the label name?

Reply:
Its a more accurate label

Reply:
Link to cell [here](https://docs.google.com/spreadsheets/d/13PHkzorfTmYRjuIJv3fCml3zBAuDba6Mgdpu4H7cAg8/edit#gid=0&range=225:225)